translib/transformer/xfmr_intf.go

```diff
@@ -302,6 +305,30 @@ var intf_table_xfmr TableXfmrFunc = func(inParams XfmrParams) ([]string, error)
 		}
 	}
 
+	if (ifName == "*") && (inParams.oper == SUBSCRIBE) {
+		log.Info("intf_table_xfmr * ifName subscribe with targetUriPath ", targetUriPath)
+
+		if strings.HasPrefix(targetUriPath, "/openconfig-interfaces:interfaces/interface/config") {
+			tblList = append(tblList, "PORT")
+		} else if strings.HasPrefix(targetUriPath, "/openconfig-interfaces:interfaces/interface/state") {
+			tblList = append(tblList, "PORT_TABLE")
```

Comment:
What about `/interfaces/interface/state/counters` path? It should have been mapped to COUNTERS, right?

Reply:
The Subscribe_intf_get_counters_xfmr handles the mapping of interfaces/interface/state/counters to COUNTERS_PORT_NAME_MAP